contrib/check-config.sh

```diff
@@ -34,6 +34,17 @@ kernelMajor="${kernelVersion%%.*}"
 kernelMinor="${kernelVersion#$kernelMajor.}"
 kernelMinor="${kernelMinor%%.*}"
 
+check_cgroup() {
+    local sys_fs_cgroup="$(grep -E '\s+/sys/fs/cgroup\s+' /proc/mounts | head -n 1 | cut -d ' ' -f1)"
```

Comment:
just a quick blurb (haven't fully reviewed);

we try to keep this script POSIX; so we should probably avoid using `local` here